docs/source/deploying/configuration.rst

```diff
@@ -122,6 +122,24 @@ Quetz can store package in object cloud storage compatible with S3 interface. To
 :bucket_prefix:
 :bucket_suffix: channel directories on S3 are created with the following semantics: ``{bucket_prefix}{channel_name}{bucket_suffix}``
 
+``redis`` section
+^^^^^^^^^^^^^^^^^^^^^^
+
+Quetz can be configured to run with a redis-server and use redis-queue for queueing jobs and processing them in the background with workers.
+Note that you might need to run ``redis-server`` and ``rq worker`` commands in separate terminals / containers along with quetz.
+You can configure the redis-server by setting the host, port and db:
+
+.. code::
+
+   [redis]
+   ip = "127.0.0.1"
+   port = 6379
+   db = 0
```

Comment:
what is db? is this an integer or string? how can we make sure that the db exist?

Reply:
The default database that `redis` uses is called `db` 0 to store stuff.